src/jti_background.js

```diff
@@ -176,15 +202,35 @@ function setDefaultTemplates (callback) {
 function downloadJSONData (url, callback) {
     fetchJSON(url, function (status, message, data) {
         if (status) {
-            data.templates = JSONtoTemplateData(data.templates);
-            data.options.domains = JSONtoDomainData(data.options.domains);
+            data = JSONtoData(data);
             saveTemplates(data, callback);
         } else {
             callback(false, message);
         }
     });
 }
 
+function loadLocalFile (fileContents, callback) {
+    try {
+        var templateJSON = JSON.parse(fileContents);
+        templateJSON = JSONtoData(templateJSON);
+        saveTemplates(templateJSON, callback);
```

Comment:
We're not actually using the variable templateJSON for anything. We can combine these three lines